book/07-git-tools/sections/reset.asc

```diff
@@ -311,5 +311,5 @@ Pay especial attention to the 'WD Safe?' column – if it says *NO*, take a seco
 | `checkout [commit]` | HEAD | YES | YES | YES
 | *File Level* | | | |
 | `reset (commit) [file]` | NO | YES | NO | YES
-| `checkout (commit) [file]` | NO | YES | YES | *NO*
+| `checkout [commit] [file]` | NO | YES | YES | *NO*
```

Comment:
🤔 According to this convention, the `file` parameter isn't optional here. Maybe change this to `checkout [commit] (file)`?

Reply:
According to our recent discussions, that would be <file>, no? The man pages take it one step further, using the syntax [<branch>] for something optional, but that's probably getting a bit carried away.

Reply:
This part of the table is about file-specific operations using `reset` and `checkout`. If you don't include a file argument, then the behavior is specified elsewhere in the table.

Reply:
Sorry, didn't realize that commenting did strange things to square and angle brackets, which is why my note looks so weird. I was only suggesting that, based on recent conversations, a non-optional argument should be contained in angle brackets, correct?

Reply:
Oh right, yes, this would match up with your changes in other chapters.

@lheckemann, can you update your PR to surround optional arguments in `[square brackets]` and required-but-replaceable arguments in `<angle brackets>`?